README.md

```diff
@@ -240,4 +240,8 @@ As of February 2021, index resources that don't support tags aren't applicable t
 May 2020: Microsoft.DocumentDB/databaseAccounts/ipRangeFilter updated from a string property to an array.  Please re-author your custom definitions to support the property as an array.  
 July 2020: The alias Microsoft.Sql/servers/securityAlertPolicies/emailAddresses[] and related policies were deprecated. 
 
+### Policy evaluating containers does not support container exclusions currently.  
```

Comment:
This README is a mismash of issues from all types of policies.  This issue needs to make it clear what policy modes this issue pertains to (i.e. just the Kubernetes modes).  Otherwise the reader won't know what the word "container" means